modules/nw-ingress-sharding.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.adoc
+// * ingress-operator.adoc
+
+[id="nw-ingress-sharding_{context}"]
+= Ingress Controller sharding
+
+As the primary mechanism for traffic to enter the cluster the demands on the Ingress Controller, or router, can be significant. As cluster administrator, you can shard the routes to:
+
+* Balance Ingress Controllers, or routers, with several routes to speed up responses to changes.
+* Allocate certain routes to have different reliability guarantees than other routes.
+* Allow certain Ingress Controllers to have different policies defined.
+* Allow only specific routes to use additional features.
+* Expose different routes on different addresses, which for example, would allow external and internal users see different routes.
```

Comment:
I would reword this slightly; "which for example" is an odd phrase, and I'd also swap external/internal:

> Expose different routes on different addresses so that internal and external users can see different routes, for example.

Reply:
Thank you! That one was bothering me too.